bottlerocket-settings-sdk/tests/migration_validation/null/mod.rs

```diff
@@ -0,0 +1,33 @@
+use super::common::define_model;
+use anyhow::Result;
+use bottlerocket_settings_sdk::{
+    extension::SettingsExtensionError, BottlerocketSetting, GenerateResult,
+    NullMigratorExtensionBuilder, SettingsModel,
+};
+use serde::{Deserialize, Serialize};
+
+define_model!(NullModelA, "v1");
+define_model!(NullModelB, "v2");
+
+#[test]
+fn test_single_model() {
+    assert!(matches!(
+        NullMigratorExtensionBuilder::with_name("null-migrator")
+            .with_models(vec![BottlerocketSetting::<NullModelA>::model()])
+            .build(),
+        Ok(_)
+    ));
```

Comment:
Or
```suggestion
    assert!(
        NullMigratorExtensionBuilder::with_name("null-migrator")
            .with_models(vec![BottlerocketSetting::<NullModelA>::model()])
            .build()
            .is_ok()
    );
```

Or (this is probably best because the Error is shown in the panic text)

```rust
        NullMigratorExtensionBuilder::with_name("null-migrator")
            .with_models(vec![BottlerocketSetting::<NullModelA>::model()])
            .build()
            .unwrap();
```